### PR TITLE
[COST-4375] delete IngressReportDBAccessor

### PR DESCRIPTION
**Pull request by maskarb**
## Jira Ticket

[COST-4375](https://issues.redhat.com/browse/COST-4375)

## Description

Eliminate the IngressReportDBAccessor and replace with ORM methods.

## Testing

1. Checkout Branch
2. Restart Koku
3. Hit endpoint or launch shell
    1. You should see ...
4. Do more things...

## Notes

...

### CODE REVIEW

File: koku/masu/processor/parquet/parquet_report_processor.py [hunk 1]
```diff
@@ -490,12 +491,12 @@ def create_parquet_table(self, parquet_file, daily=False, partition_map=None):
 
     def check_required_columns_for_ingress_reports(self, post_processor, col_names):
         LOG.info(log_json(msg="checking required columns for ingress reports", context=self._context))
-        if not check_ingress_columns:
+        if not check_ingress_columns(self.schema_name):
```
Comment (maskarb) [inline koku/masu/processor/parquet/parquet_report_processor.py:494]: This if statement was never correct. Do we even need it?

File: koku/masu/processor/parquet/parquet_report_processor.py [hunk 2]
```diff
@@ -274,52 +291,45 @@ def parquet_file_getter(self):
...
-        post_processor = None
-        if self.provider_type in [Provider.PROVIDER_AWS, Provider.PROVIDER_AWS_LOCAL]:
-            post_processor = AWSPostProcessor(schema=self._schema_name)
-        elif self.provider_type in [Provider.PROVIDER_AZURE, Provider.PROVIDER_AZURE_LOCAL]:
-            post_processor = AzurePostProcessor(schema=self.schema_name)
-        elif self.provider_type in [Provider.PROVIDER_GCP, Provider.PROVIDER_GCP_LOCAL]:
-            post_processor = GCPPostProcessor(schema=self._schema_name)
-        elif self.provider_type in [Provider.PROVIDER_OCI, Provider.PROVIDER_OCI_LOCAL]:
-            post_processor = OCIPostProcessor(schema=self._schema_name)
+        if self.provider_type == Provider.PROVIDER_AWS:
```
Comment (maskarb) [inline koku/masu/processor/parquet/parquet_report_processor.py:297]: The `self.provider_type` property strips the `-local` so we don't need to compare all of these with the `_LOCAL` provider types.

**Review by lcouzens — APPROVED** [review]
LGTM

**Pull request merged**